.github/workflows/release-embedding-sdk.yml

```diff
@@ -171,13 +164,6 @@ jobs:
           git config --global user.email "github-automation@metabase.com"
           git config --global user.name "Metabase Automation"
 
-      # TODO: Remove this step after the build utils script is merged to master
```

Comment:
Same as the previous comment. The previous step updates the util and this step downloads the util